content/terraform-docs-common/docs/docs/tools/mcp-server/deploy.mdx

```diff
@@ -6,20 +6,30 @@ description: |-
 
 # Deploy the Terraform MCP server
 
-This topic describes how to deploy the Terraform model context protocol (MCP) server in a Docker container on your local workstation. The Terraform MCP server connects to a client so that your LLM can use data stored in the Terraform registry to help you write Terraform configuration. Refer to the [overview](/terraform/docs/terraform-tools/mcp-server) for additional information about the Terraform MCP server.
+The Terraform MCP server runs in a Docker container that you can deploy to either your local workstation or a remote instance. When running the server locally, start it in the default `stdio` mode. In this mode, the server receives and sends messages in JSON format over RPC. When running the server on a remote instance, start the server in `streamable-http` mode so that you can communicate with the server using the Streamable HTTP transport.
```

Comment:
```suggestion
The Terraform MCP server runs in a Docker container that you can deploy to either your local workstation or a remote instance. When running the server locally, start it in the default `stdio` mode. In this mode, the server communicates over standard input/output. When running the server on a remote instance, start the server in `streamable-http` mode so that you can communicate with the server using the Streamable HTTP transport.
```